tests/test_athena.py

```diff
@@ -595,6 +595,30 @@ def test_read_sql_query_wo_results(path, glue_database, glue_table):
     ensure_athena_query_metadata(df=df, ctas_approach=False, encrypted=False)
 
 
+def test_read_sql_query_wo_results_chunked(path, glue_database, glue_table):
```

Comment:
Looks good, is it worth refactoring -> consolidating these functions into one, adding `ctas_approach` as an argument?

Reply:
Great point, I'll do that.